src/libraries/Common/src/System/Net/NetworkInformation/UnixCommandLinePing.cs

```diff
@@ -9,8 +9,8 @@ namespace System.Net.NetworkInformation
 {
     internal static class UnixCommandLinePing
     {
-        // Ubuntu has ping under /bin, OSX under /sbin, ArchLinux under /usr/bin, Android under /system/bin.
-        private static readonly string[] s_binFolders = { "/bin/", "/sbin/", "/usr/bin/", "/system/bin" };
+        // Ubuntu has ping under /bin, OSX under /sbin, ArchLinux under /usr/bin, Android under /system/bin, NixOS under /run/current-system/sw/bin.
+        private static readonly string[] s_binFolders = { "/bin/", "/sbin/", "/usr/bin/", "/system/bin", "/run/current-system/sw/bin/" };
```

Comment:
```suggestion
        private static readonly string[] s_binFolders = { "/bin", "/sbin", "/usr/bin", "/system/bin", "/run/current-system/sw/bin" };
```

Reply:
Path.Combine handles perfectly the cases with / at the end and without

Reply:
trailing slash isn't necessary, matching /system/bin, goes easy on eyes